### DISCUSSION TITLE
How to know the path of the file in GitHub Actions

### DISCUSSION THREAD

**Discussion by keshav12-deloitte** [community/community#39725] [New to GitHub]
Hey Hi Everyone ,I am new to Github Actions i want to run my python file which is in sub Folder Called Teste ,can anyone help me how can i run the name.py file which is present in Folder called Tests .Any Answer related to Path is Appreciated.
![Screenshot (322)](https://user-images.githubusercontent.com/101085487/203246851-75899da3-f764-4afa-b1ae-1c5fa7e5025e.png)

**Comment by airtower-luna** [comment]
After checking out the repository it'll be available under its path inside the repository, like `Tests/name.py`. When asking for help with Actions it's generally best if you link a log of the workflow you're having trouble with, so people can take a look.
